src/KafkaFlow.OpenTelemetry/ActivitySourceAccessor.cs

```diff
@@ -21,9 +21,10 @@ internal static class ActivitySourceAccessor
         internal static readonly string Version = Assembly.GetExecutingAssembly().GetName().Version.ToString();
         internal static readonly ActivitySource ActivitySource = new(ActivitySourceName, Version);
 
-        public static void SetGenericTags(Activity activity)
+        public static void SetGenericTags(Activity activity, IEnumerable<string> bootstrapServers)
```

Comment:
Is it possible to receive null value for the bootstrapServers param?

Reply:
Unlikely as the bootstrapServers param comes from the ClusterConfiguration. If those values are null, the cluster is not set up at all

Reply:
Can you check if the string.join will throw exception if the bootstrapServers is null?